### PR TITLE
Fix missing attribute when convert with multiple docker-compose files

### PR DESCRIPTION
**Pull request by xianlubird**
Fix issue #972 and #968

A brief introdction:
- When multiple docker compose files are passed, we should parse every compose file and override previous object if next compose file has this field
- Docker compose merge multiple file, so we should do so
- This function is very useful
- First, I implemented the v3 compose version. Please @cdrage @hangyan help me review. If you guys accept this method, I will do the same thing to v2 compose version.

Thanks .

Signed-off-by: xianlubird <xianlubird@gmail.com>

### CODE REVIEW

**Comment by xianlubird** [comment]
@hangyan Done. Thanks

**Comment by xianlubird** [comment]
Any update @cdrage ?

**Comment by cdrage** [comment]
I'll be able to do a few reviews next week, could you do a review @hangyan ?

**Comment by hangyan** [comment]
Sure, will review this in a few days

?? Outlook for Android<https://aka.ms/ghei36>

________________________________

File: pkg/loader/compose/v3.go [hunk 1]
```diff
@@ -66,44 +67,55 @@ func parseV3(files []string) (kobject.KomposeObject, error) {
...
+			WorkingDir:  workingDir,
+			ConfigFiles: []types.ConfigFile{configFile},
+			Environment: env,
+		}
+
+		// Actual config
+		// We load it in order to retrieve the parsed output configuration!
+		// This will output a github.com/docker/cli ServiceConfig
+		// Which is similar to our version of ServiceConfig
+		tmpConfig, err := loader.Load(configDetails)
```
[the remark this answers is not in the document]
Comment (xianlubird) [reply pkg/loader/compose/v3.go]: @hangyan Thank you for your review. I will refactor code ASAP.

**Comment by xianlubird** [comment]
@hangyan I have factor code:

For some questions:

1. We also use kompose convert -f base.yml -f prod.yml ,  so that's match.

2. I have fixed this.

3. Already add openshift test case

Pls review again, if you have time. Thanks.

**Comment by xianlubird** [comment]
@hangyan I see #997 was merged, Pls review it again, Thank you.

File: script/test/cmd/tests.sh [hunk 2]
```diff
@@ -122,7 +122,23 @@ cmd="kompose -f $KOMPOSE_ROOT/script/test/fixtures/mem-limit/docker-compose-mb.y
...
+sed -e "s;%VERSION%;$version;g" -e "s;%CMD%;$cmd;g"  $KOMPOSE_ROOT/script/test/fixtures/merge-multiple-compose/output-compose-port-template.json > /tmp/output-k8s.json
+convert::expect_success "$cmd" "/tmp/output-k8s.json"
 
+cmd="kompose convert -f $KOMPOSE_ROOT/script/test/fixtures/merge-multiple-compose/compose-port-base.yml -f $KOMPOSE_ROOT/script/test/fixtures/merge-multiple-compose/compose-new-service-prob.yml --stdout -j"
+sed -e "s;%VERSION%;$version;g" -e "s;%CMD%;$cmd;g"  $KOMPOSE_ROOT/script/test/fixtures/merge-multiple-compose/output-compose-new-service-template.json > /tmp/output-k8s.json
+convert::expect_success "$cmd" "/tmp/output-k8s.json"
+
+cmd="kompose --provider=openshift convert -f $KOMPOSE_ROOT/script/test/fixtures/merge-multiple-compose/compose-port-base.yml -f $KOMPOSE_ROOT/script/test/fixtures/merge-multiple-compose/compose-new-service-prob.yml --stdout -j"
+sed -e "s;%VERSION%;$version;g" -e "s;%CMD%;$cmd;g"  $KOMPOSE_ROOT/script/test/fixtures/merge-multiple-compose/output-openshift-template.json > /tmp/output-k8s.json
```
Comment (hangyan) [inline script/test/cmd/tests.sh]: `output-k8s.json` -> `output-os.json`, which is more straightforward

**Comment by xianlubird** [comment]
@hangyan Done.

**Comment by hangyan** [comment]
Thanks

**Pull request merged**